tools/rna_tools/coprarna/coprarna.xml

```diff
@@ -5,14 +5,18 @@
   <requirements>
     <requirement type="package" version="2.1.1">coprarna</requirement>
   </requirements>
+  <stdio>
+    <exit_code range="1:" level="fatal" description="Failing exit code."/>
+    <exit_code range=":-1" level="fatal" description="Failing exit code."/>
+  </stdio>
   <command detect_errors="aggressive"><![CDATA[
```

Comment:
if you use detect_errors you can remove the stdio tag, at least in this case how you use it